webpack.config.js

```diff
@@ -69,7 +69,15 @@ module.exports = {
           contextPath: '\${util.ConfigHelper.contextPath}',
           jsSource: `\${resource(dir: '${path.basename(WEBPACK_OUTPUT)}', file: 'bundle.${compilation.hash}.js')}`,
           cssSource: `\${resource(dir: '${path.basename(WEBPACK_OUTPUT)}', file: 'bundle.${compilation.hash}.css')}`,
-          receivingIfStatement: '',
+          receivingIfStatement:
```

Comment:
This should not be injected like that. If you'd like to inject it through `templateParameters` it should have its own parameter instead of using the of related to receiving. However, you correctly found the reason behind it, but missed one key part in your video. The issue is here:

Develop's `grails-template.html`::
https://github.com/openboxes/openboxes/blob/develop/src/assets/grails-template.html#L4

OBGM feature branch's `grails-template.html`::
https://github.com/openboxes/openboxes/blob/feature/upgrade-to-grails-3.3.10/src/assets/grails-template.html#L4

Develop's `main.gsp`:
https://github.com/openboxes/openboxes/blob/develop/grails-app/views/layouts/main.gsp

Feature branch's `main.gsp`, and `react.gsp`:
https://github.com/openboxes/openboxes/blob/feature/upgrade-to-grails-3.3.10/grails-app/views/layouts/main.gsp
https://github.com/openboxes/openboxes/blob/feature/upgrade-to-grails-3.3.10/grails-app/views/layouts/react.gsp

So it's better to fix the layout used in the `grails-template.html` than additionally injecting it by webpack.

Reply:
@kchelstowski looks like react pages don't load layout properly

Reply:
@kchelstowski if you did not get any progress with layout rendering, add a new template param, that will inject into header entire content of the header from `main.gsp` and please create a ticket to investigate the layouts rendering on react.